src/main/java/teammates/common/util/Const.java

```diff
@@ -1057,7 +1057,6 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String MASHUP = "/mashup.jsp";
     
         //View fragments
-        public static final String FOOTER = "/jsp/footer.jsp";
         public static final String FEEDBACK_SUBMISSION_EDIT = "/jsp/feedbackSubmissionEdit.jsp";
```

Comment:
@wkurniawan07 Should the `feedbackSubmissionEdit.jsp` be a `.tag` instead? Weird that it is the only remaining view fragment.